bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/services/s3/S3BagVerifier.scala

```diff
@@ -54,26 +55,32 @@ trait S3BagVerifier[B <: BagVerifyContext[S3ObjectLocationPrefix]]
     location.key.replace(root.keyPrefix, "")
 }
 
-class S3StandaloneBagVerifier(val primaryBucket: String)(
+class S3StandaloneBagVerifier(val bucket: String)(
   implicit val s3Client: AmazonS3
-) extends StandaloneBagVerifier[S3ObjectLocation, S3ObjectLocationPrefix]
-    with S3BagVerifier[StandaloneBagVerifyContext[S3ObjectLocationPrefix]]
+) extends BagVerifier[
+      StandaloneBagVerifyContext[S3ObjectLocationPrefix],
+      S3ObjectLocation,
+      S3ObjectLocationPrefix
+    ]
+    with S3BagVerifier[StandaloneBagVerifyContext[S3ObjectLocationPrefix]] {
+  override def verifyReplicatedBag(
+    root: StandaloneBagVerifyContext[S3ObjectLocationPrefix],
+    space: StorageSpace,
+    externalIdentifier: ExternalIdentifier,
+    bag: Bag
+  ): Either[BagVerifierError, Unit] = Right(())
```

Comment:
Can we delete this now?

Reply:
I don't think so, It needs to be implemented as a no-op thing somewhere. Either here or in the base `BagVerifier` trait. I prefer here cause it means the `BagVerifier` makes fewer assumptions but ..meh